src/commands/Management/createMute.ts

```diff
@@ -15,7 +17,7 @@ export default class extends SkyraCommand {
 			description: language => language.tget('COMMAND_CREATEMUTE_DESCRIPTION'),
 			extendedHelp: language => language.tget('COMMAND_CREATEMUTE_EXTENDED'),
 			permissionLevel: PermissionLevels.Administrator,
-			requiredGuildPermissions: ['MANAGE_CHANNELS', 'MANAGE_ROLES'],
+			requiredGuildPermissions: ['MANAGE_CHANNELS', 'MANAGE_ROLES', 'USE_EXTERNAL_EMOJIS', 'ADD_REACTIONS', 'READ_MESSAGE_HISTORY'],
```

Comment:
Those permissions are guild-wide, Skyra might have them revoked specifically in the channel, or granted.

What I meant was to add `if (message.reactable) await message.react(...);`, maybe with a text branch as fallback.

Reply:
Done